site/docs/overview/styling.md

```diff
@@ -116,6 +116,151 @@ When processing your code into CSS, vanilla-extract will always render your medi
 
 > 🧠&nbsp;&nbsp;When it's safe to do so, vanilla-extract will merge your `@media`, `@supports`, and `@container` condition blocks together to create the smallest possible CSS output.
 
+## Layers
```

Comment:
This feels a bit out of step with the rest of the sections on this page. I think we need to add at least a little description to explain what's going on in the code.

Reply:
100% agree this needs more. It was just a placeholder to showcase the capabilities.